tox.ini

```diff
@@ -0,0 +1,113 @@
+[tox]
```

Comment:
I'm OK with this for now, but I'm not sure if I want to use tox or nox longer term, so I consider it as tentative for the moment. In principle I'd have gone for something simpler, but I don't see any value in trying to simplify this *just* because you've provided something more complex than I'd have started with 🙂